### ISSUE TITLE
Be able to switch monitor

### ISSUE THREAD

**Issue by EmberLightVFX** [tyami94/RemoteDesktop#1]
Greate little software this! One bonus would be to be able to switch monitors, or maybe even be able to view multiple monitors.
There could be another button in the menu (stream desktop/exit) to switch to the next window.
I'm guessing this sofware is just a transparent window that could be moved to other monitors?

**Comment by tyami94** [comment]
I had totally forgotten about this program until today. It is super neat to see that people are liking my (admittedly janky) software. You have the right idea. This is just a window that makes the Steam client think you're playing a game. I ran into a situation that needed near-realtime remote access, and I figured the simplest solution is generally the best one.

Unfortunately, my solution may not work for you. Multiple monitor support can't really be implemented here as far as I know. I have multiple monitors, and I could only ever get it to work on one. It would have to be a limitation of the Steam client. There may be another solution that supports this, but I'm not really sure.

If anyone can tell me otherwise, I'd get right on to adding it for you.

--Tyler

**Comment by atk3001** [comment]
I moved monitors in steam link via teamviewer. You just have to move the mouse to another screen. So you could maybe implement a hotkey or button to put the mouse on another screen.

**Comment by Sidewardduke** [comment]
Yeah same, that's typically how I switch monitors. It would be very useful to have as I use this program anytime I'm not able to use TeamViewer or their servers are down.

**Comment by Krubbs** [comment]
Hey guys, check this simple UI improvement. It will change mouse position for one screen left or right.
[RemoteDesktop.zip](https://github.com/tyami94/RemoteDesktop/files/5898541/RemoteDesktop.zip)
